src/components/App/App.js

```diff
@@ -101,10 +103,29 @@ const App = () => {
 
   return (
     <div className="App">
-      <Header />
-      <AllBreweries allBreweriesData={allBreweries} updateFavorites={updateFavorites} favoritesList={favorites}/>
 
-      <Favorites favoritesList={favorites} updateFavorites={updateFavorites}/>
+      <Route exact path='/'
```

Comment:
I would eventually wrap all routers with Switch, so it will check matching path correctness, overall your setUp looks great